### DISCUSSION TITLE
✅ Allow to Select Site Content (pages, posts, etc.) from a dropdown in the Button widget's Link field

### DISCUSSION THREAD

**Discussion by SSL-INC** [elementor/elementor#27598] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

Time saving.

### Describe the solution you'd like.

Using the Elementor button widget, it would be nice to have a dropdown menu when liking to another page, just like in the old days. I hate having to manually type in a URL every time for the button links. I feel like we have gone backward in time. I used to be so much better before on WPBakery and Visual Composer.

### Describe alternatives you've considered.

Give us a list of internal pages we can link to within a dropdown menu.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by nicholaszein** [comment]
Hi @SSL-INC!

This is already possible with Elementor! Start typing the name of the Post or Page you want to link to, and a list will appear.

Alternatively, if you have Elementor Pro, you can use the **`Dynamic Tag`** option in the link field, and select the option **`Internal URL`**, then start typing the name of the Post or Page to use it as dynamic link. The difference here is there if the URL of the destination link changes, it will be automatically changed in the Button link as well.

This is covered in our [Help Center](https://elemn.to/help) and [Academy](https://elemn.to/academy).

I'm closing this Request as it's already an existing feature since Elementor 2.0.

Kind regards
